pkg/client/password_test.go

```diff
@@ -0,0 +1,21 @@
+package client
+
+import (
+	"testing"
+)
+
+func TestCheckPassword(t *testing.T) {
+	var testCases = []struct {
+		pass string
+		ok   bool
+	}{
+		{"12345678", true},
+		{"test", false},
+	}
+
+	for _, tc := range testCases {
+		if err := checkPassword(tc.pass); err != nil && tc.ok {
+			t.Errorf("expected failure, got success")
```

Comment:
@drgarcia1986 updated, I think the original logic is wrong, we have to check both cases.